ols/app/metrics.py

```diff
@@ -10,11 +10,18 @@
     "response_duration_seconds", "Response durations", ["path"]
 )
 
-llm_calls_total = Counter("llm_calls_total", "LLM calls counter")
+llm_calls_total = Counter("llm_calls_total", "LLM calls counter", ["provider", "model"])
 llm_calls_failures_total = Counter("llm_calls_failures_total", "LLM calls failures")
 llm_calls_validation_errors_total = Counter(
     "llm_validation_errors_total", "LLM validation errors"
 )
 
+llm_token_sent_total = Counter(
```

Comment:
is Counter the best option? Probably histogram might be better fit?

Reply:
Couter is the cheapest option to meet the needs for [OLS-159](https://issues.redhat.com/browse/OLS-159) :) 
Of course the histogram give us more insights on the token count distribution in its `_buckets` metrics. 
Do we need these details about token count? @bparees ?
Available metrics types: https://prometheus.io/docs/tutorials/understanding_metric_types/

Reply:
as discussed in standup meeting, we keep using a simple Counter for the moment.